packages/flame_splash_screen/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+## [Next]
```

Comment:
is this a melos-compatible format?

Reply:
Definitely not :sweat_smile:

Reply:
should we restart?

Reply:
I'll remove this entry and then write a manual entry for the next version we release, since there aren't any conventional commits connected to it.